pf/tests/integration/program_test.go

```diff
@@ -24,17 +24,19 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
-
 	"github.com/pulumi/pulumi/pkg/v3/engine"
 	"github.com/pulumi/pulumi/pkg/v3/testing/integration"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"sourcegraph.com/sourcegraph/appdash"
 
 	"github.com/pulumi/pulumi-terraform-bridge/v3/unstable/testutil"
 )
 
 func TestBasicProgram(t *testing.T) {
+	if testing.Short() {
```

Comment:
I'm not sure we have short and long tests in the bridge - are these needed?

Reply:
I noticed we have different `test.integration` and `test.unit` targets, and `-short` is an easy way to exclude slow integration tests from unit tests (without needing to resort to things like build tags, modules, etc.).